### ISSUE TITLE
Persist breakpoints across sessions?

### ISSUE THREAD

**Issue by mightyiam** [jaridmargolin/inspect-process#30]
I find myself having to set breakpoints all over again, after I "lose" my bug and the execution terminates.

**Comment by jaridmargolin** [comment]
I agree this is less than ideal. I think the problem could be solved with a mechanism to "restart" the process.

**Comment by jaridmargolin** [comment]
For reference: https://github.com/eugeneo/node-restarter
